metrics/core/build.gradle

```diff
@@ -38,11 +38,12 @@ dependencies {
   implementation project(':plugin-api')
 
   implementation 'com.google.guava:guava'
-  implementation 'io.grpc:grpc-netty'
+  implementation 'io.grpc:grpc-netty-shaded'
```

Comment:
Again, is the shaded version really necessary? I have concerns about proper security auditing.